homeassistant/components/matter/strings.json

```diff
@@ -252,6 +255,11 @@
         "name": "Power"
       }
     },
+    "vacuum": {
+      "vacuum": {
+        "name": "[%key:component::vacuum::title%]"
+      }
+    },
```

Comment:
Its future proofed. It will use None if a single vacuum is discovered but if multiple vacuums get discovered (e.g. in case of a bridge) it needs to fallback to this string

Reply:
But if its a bridge, will the bridge be the device? I would assume it would create a separate device for the physical device

Reply:
Yeah, bridge was a bad example. A composed device with multiple vacuum endpoints.
Its a bit unlikely but never say never with Matter :-)

At least we have the same consistent pattern across platforms now. As you can see the name is by default set to None to use the device name and it will only fallback to the 'platform translation key' if there are multiple endpoints.